odoo/addons/base/tests/test_res_users.py

```diff
@@ -295,23 +295,6 @@ def test_change_user_login(self):
                 "Setting a valid email as login should update the partner's email"
             )
 
-    def test_reified_groups(self):
```

Comment:
Why didn't you adapt the test, instead of removing it?

Reply:
Eventually I could test with the default group. It is absolutely necessary to change the name of the test which no longer means anything.